### DISCUSSION TITLE
Add Arctic Fuse 2 Theme Support

### DISCUSSION THREAD

**Discussion by The-Crow-Den** [Goldenfreddy0703/Otaku#344] [Ideas]
Will Arctic Fuse 2 support be added since Arctic Fuse 1 themes don't function on Arctic Fuse 2?

**Comment by Goldenfreddy0703** [comment]
Well crap, i thought Arctic Fuse 1 themes were supported and all cause its Arctic Fuse of course and i really did not want to make a theme for Arctic Fuse 2 as well as a theme for Seren. Will looks into at some point this week and see what i can do. Thank you

**Reply by The-Crow-Den** [reply]
No thank you for your hard work

**Comment by Goldenfreddy0703** [comment]
Hey so we added Theme support
